content/millennium/dstu2/medications/medication-administration.md

```diff
@@ -160,21 +157,15 @@ _Implementation Notes_
 <%= headers status: 200 %>
 <%= json(:dstu2_medication_administration_multiple_ingredients_entry) %>
 
-<%= disclaimer %>
-
-### Errors
-
-The common [errors] and [OperationOutcomes] may be returned.
-
 [`reference`]: http://hl7.org/fhir/DSTU2/search.html#reference
 [`token`]: http://hl7.org/fhir/DSTU2/search.html#token
 [`date`]: http://hl7.org/fhir/DSTU2/search.html#date
 [`quantity`]: http://hl7.org/fhir/DSTU2/search.html#quantity
 [`_count`]: http://hl7.org/fhir/DSTU2/search.html#count
 [`number`]: http://hl7.org/fhir/DSTU2/search.html#number
-[`CodeableConcept`]: http://hl7.org/fhir/DSTU2/datatypes.html#codeableconcept
+[CodeableConcept]: http://hl7.org/fhir/DSTU2/datatypes.html#codeableconcept
 [contained]: http://hl7.org/fhir/DSTU2/references.html#contained
-[MedicationAdministration.medication]: http://hl7.org/fhir/DSTU2/medicationadministration-definitions.html#MedicationAdministration.medication_x_
 [errors]: ../../#client-errors
 [OperationOutcomes]: ../../#operation-outcomes
 [Infuse Over Time]: #custom-extensions
```

Comment:
so, this is a link only used once and it's a reference to the line below it? unless I'm missing something I do see that the list of fields mentions it, but it has its own link to the higher-level Extensions header.